test/Event/FileWriter/TabularDataEventFormatterTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+
+namespace CultuurNet\UDB3\EventExport\FileWriter;
+
+use CultuurNet\UDB3\EventExport\FileWriter\TabularDataEventFormatter;
+use CultuurNet\UDB3\EventExport\FileWriter\TabularDataFileWriter;
+
+class TabularDataEventFormatterTest extends \PHPUnit_Framework_TestCase
+{
+
+    private function getJSONEventFromFile($fileName)
+    {
+        $jsonEvent = file_get_contents(
+            __DIR__ . '/' . $fileName
+        );
+
+        return $jsonEvent;
+    }
+
+    /**
+     * @test
+     */
+    public function it_excludes_all_terms_when_none_are_included()
+    {
+        $includedProperties = [
+            'id',
+            'name',
+            'calendarSummary',
+            'image',
+            'location',
+            'address',
+            'bookingInfo'
+        ];
+        $eventWithTerms = $this->getJSONEventFromFile('event_with_terms.json');
+        $formatter = new TabularDataEventFormatter(
+            TabularDataFileWriter::columns(),
+            $includedProperties
+        );
+
+        $formattedEvent = $formatter->formatEvent($eventWithTerms);
+
+        $this->assertArrayNotHasKey('terms.theme', $formattedEvent);
+        $this->assertArrayNotHasKey('terms.eventtype', $formattedEvent);
```

Comment:
Better to do an exact match on the keys you expect to be returned instead of a negative assertion.